contrib/docker/nightly-tester/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:22.04
+
+LABEL maintainer <rene.gassmoeller@mailbox.org>
+
+RUN DEBIAN_FRONTEND=noninteractive apt update && apt upgrade -yq && \
+  DEBIAN_FRONTEND=noninteractive apt install -yq --no-install-recommends \
+  build-essential ca-certificates file gcc g++ \
+  gfortran git libblas-dev liblapack-dev libopenmpi-dev \
+  lsb-release ninja-build numdiff openmpi-bin \
+  openmpi-common wget zlib1g-dev \
+  texlive-plain-generic texlive-base texlive-latex-recommended \
+  texlive-latex-base texlive-fonts-recommended \
```

Comment:
should we delete the latex stuff?

Reply:
Do we even still need it? We used to need it for the manual.